sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -566,7 +565,7 @@ def _rows_to_fqt_events(self, rows):
             if query_cache_key in self._full_statement_text_cache:
                 continue
             self._full_statement_text_cache[query_cache_key] = True
-            tags = list(self.tags)
+            tags = copy.copy(self._check.tag_manager.get_tags())
```

Comment:
This copy isn't needed. `TagManager.get_tags()` returns a [mutation free list](https://github.com/DataDog/integrations-core/blob/f562e5a2fed5ec3cb212acf6da9fcd7f25d4660a/datadog_checks_base/datadog_checks/base/utils/db/utils.py#L569)